File: src/main/java/com/proyecto/macrofit/rutinaejercicio/model/RutinaUsuario.java
package com.proyecto.macrofit.rutinaejercicio.model;

import java.time.LocalDate;

import lombok.Data;

@Data
public class RutinaUsuario {

    private Integer idRutinaUsuario;
    private Integer idRutina;
    private Integer idUsuario;
    private LocalDate fechaInicio;
    private LocalDate fechaFin;
    private Boolean activo;

    
}

File: Producto/MacroFit/app/src/main/java/com/duoc/macrofit/rutinas/model/RutinaUsuario.kt
package com.duoc.macrofit.rutinas.model

import com.google.gson.annotations.SerializedName

data class RutinaUsuario(
    val idRutinaUsuario: Int,

    val idRutina: Int,

    val idUsuario: Int,

    val fechaInicio: String?,

    val fechaFin: String?,

    val activo: Boolean?
)